pages/components/alert/alert.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="../../../styles.css" />
+		<title>Alert | UI Game</title>
+	</head>
+	<body>
+		<nav>
+			<header class="navbar-heading">UI Game</header>
+		</nav>
+
+		<hr />
+
+		<div class="main-body">
+			<h1>Alert</h1>
+			<p>
+				Alerts are used to notify the user of an important event. They can be used to inform the user of an event, or to prompt the user to
+				take an action.
+			</p>
+
+			<h1>Error alert</h1>
+			<p>Error alerts are used to notify the user of an error.</p>
+
+			<div class="alert alert-error">
+				<svg
+					xmlns="http://www.w3.org/2000/svg"
+					xmlns:xlink="http://www.w3.org/1999/xlink"
+					aria-hidden="true"
+					role="img"
+					class="iconify iconify--mdi"
+					width="32"
+					height="32"
+					preserveAspectRatio="xMidYMid meet"
+					viewBox="0 0 24 24"
+				>
+					<path
+						d="M23 12l-2.44-2.78l.34-3.68l-3.61-.82l-1.89-3.18L12 3L8.6 1.54L6.71 4.72l-3.61.81l.34 3.68L1 12l2.44 2.78l-.34 3.69l3.61.82l1.89 3.18L12 21l3.4 1.46l1.89-3.18l3.61-.82l-.34-3.68L23 12m-10 5h-2v-2h2v2m0-4h-2V7h2v6z"
+						fill="currentColor"
+					></path>
+				</svg>
+				<p>This is a error alert</p>
+			</div>
+
+			<h1>Success alert</h1>
+			<p>Success alerts are used to notify the user of a successful event.</p>
+
+			<div class="alert alert-success">
+				<svg
+					xmlns="http://www.w3.org/2000/svg"
+					xmlns:xlink="http://www.w3.org/1999/xlink"
+					aria-hidden="true"
+					role="img"
+					class="iconify iconify--mdi"
+					width="32"
+					height="32"
+					preserveAspectRatio="xMidYMid meet"
+					viewBox="0 0 24 24"
+				>
+					<path
+						d="M23 12l-2.4-2.8l.3-3.7l-3.6-.8l-1.9-3.2L12 3L8.6 1.5L6.7 4.7l-3.6.8l.3 3.7L1 12l2.4 2.8l-.3 3.7l3.6.8l1.9 3.2L12 21l3.4 1.5l1.9-3.2l3.6-.8l-.3-3.7L23 12m-4.3 4.9l-2.7.6l-1.4 2.4l-2.6-1.1l-2.6 1.1L8 17.5l-2.7-.6l.2-2.8L3.7 12l1.8-2.1l-.2-2.8L8 6.5l1.4-2.4L12 5.2l2.6-1.1L16 6.5l2.7.6l-.2 2.8l1.8 2.1l-1.8 2.1l.2 2.8m-2.1-9.3L18 9l-8 8l-4-4l1.4-1.4l2.6 2.6l6.6-6.6z"
+						fill="currentColor"
+					></path>
+				</svg>
```

Comment:
Just a suggestion :
Inline SVGs have their advantages but,
if you're not accessing any internal property of the SVG
(eg. you want to access the fill property of SVG and change its color on hover)
I suggest using individual SVG files & importing them using an img / background tag. 
This way HTML gets shorter and a little cleaner.
Correct me if I'm wrong on this.

Reply:
Yes, you are correct. The HTML part of the project is just used for the visual representation and debugging so that's why I not using a file for SVG. Should I merge the branch if it looks good? if any suggestions do let me know.

Reply:
Ok, now I get it.
Yup, apart from this everything looks awesome.
Go ahead 👍!